### PR TITLE
String Export 2017-08-16

### PR DESCRIPTION
**Pull request by pocmo**
Those are the new strings for the Android Burgundy release.

We are past the String Freeze already. :( Finalizing those strings took longer than anticipated. Sorry, for that. Let's find a new time for the L10N Sign-off in today's cross-functional meeting. There's a whole week between submission of the final build and releasing the build. I think for the Android project it would be absolutely fine to use some of this time to catch up on translations. Especially because on Android we do not need to wait for a long time until our app is approved to be released. So we can build and submit quite late.

### CODE REVIEW

File: locales/templates/app.pot [hunk 1]
```diff
@@ -301,6 +301,26 @@ msgctxt "content_description_erase"
...
+#. button shows (expands) the menu for switching       tabs, adding a new tab
+#. or erasing the browsing history.
+msgctxt "content_description_fab_expand"
+msgid "Expand"
+msgstr ""
+
+#. Content description (not visible, for screen readers etc.): "Plus" button
+#. for adding a new     temporary tab.
+msgctxt "content_description_open_tab"
+msgid "Open tab"
```
Comment (flodolo) [inline locales/templates/app.pot]: This needs a better comment

> Description of the "+" button used to open a new temporary tab ("open" is a verb), not displayed and read only by screenreaders.

File: locales/templates/app.pot [hunk 2]
```diff
@@ -301,6 +301,26 @@ msgctxt "content_description_erase"
...
+msgstr ""
+
+#. Content description (not visible, for screen readers etc.): "X" button for
+#. closing the      screen to enter a URL for a new tab to be opened.
+msgctxt "content_description_close_tab"
+msgid "Close tab"
```
Comment (flodolo) [inline locales/templates/app.pot]: Same here, better comment (I honestly don't understand the current one at all…).

> Description of the "X" button used to close a new temporary tab ("close" is a verb), not displayed and read only by screenreaders.

File: locales/templates/app.pot [hunk 3]
```diff
@@ -581,45 +606,56 @@ msgid ""
...
-#. First run tour (page 2): Text, %1$s will be replaced with the name of the
-#. app (e.g. Firefox Focus).
-#, c-format
-msgctxt "firstrun_defaultbrowser_text"
+#. First run tour (Search): Text
+msgctxt "firstrun_search_text"
 msgid ""
-"Set %1$s as your default browser and get the benefits of private browsing"
-" when you link to the Web from other Android apps."
+"Search for something different? Choose another default search engine in "
```
Comment (flodolo) [inline locales/templates/app.pot]: Should this really by "Search" and not "Searching"?

**Review by flodolo — COMMENTED** [review]
Just a quick run through. Note that Delphine is traveling back from a localization event, and I'll be offline for almost 48 hours shortly (same reason).

**Comment by pocmo** [comment]
Thank you for the review @flodolo. I'll update the descriptions.

File: locales/templates/app.pot (hunk 1 above)
Comment (pocmo) [reply locales/templates/app.pot]: Done!

File: locales/templates/app.pot (hunk 2 above)
Comment (pocmo) [reply locales/templates/app.pot]: Done!

**Comment by pocmo** [comment]
New version pushed.

**Review by flodolo — APPROVED** [review]
I can't spot anything wrong with this, but please wait for Delphine to sync about merge and communications.

**Review by Delphine — APPROVED** [review]
This looks good, thanks Sebastian (and thanks flod for helping out while I'm traveling! :D). I'll send communication out and merge. 
@pocmo: I'm going to board soon, so I won't be able to attend the meeting as I'll be up in the air. Do I understand correctly that we could do an l10n sign-off for these one week from now? Would next Wed/Thur be OK? (we can follow-up by email if needed).
thanks!

**Comment by pocmo** [comment]
The planned sign-off date is/was Aug 24, 2017. This would already be one week. Originally we planned to have 2 weeks time for translations: https://wiki.mozilla.org/Mobile/Focus/Android/Train_Schedule - But we failed providing the strings early enough for that. Planned release date is Aug 30, 2017. So we could add a couple of more days if one week is not enough.

**Comment by Delphine** [comment]
If we could aim for an l10n hand-off Friday August 25 - or even Mon August 28 - I think it should be enough (as usual, we'd just need to be sure to get a few round of screenshots in so localizers, and I, can test their work).

**Pull request merged**